_posts/2024-01-24-practical_zephyr_kconfig.md

```diff
@@ -166,17 +165,16 @@ Message in a bottle.
 
 Huh. Even though we've disabled `PRINTK` using `CONFIG_PRINTK=n` in our `prj.conf` file, the output is still not disabled. The build passed, and there were no warnings - did we miss something?
 
-
-
 ## Navigating Kconfig
 
 As mentioned before, _Kconfig_ uses an entire _hierarchy_ of `Kconfig` files. The initial configuration is merged from several `Kconfig` files, including the `Kconfig` file of the specified board. The exact procedure used by Zephyr for merging the configuration file is explained in great detail in a [dedicated section of the official documentation](https://docs.zephyrproject.org/latest/build/kconfig/setting.html#the-initial-configuration) but is not worth repeating here.
 
 > **Notice:** The command line output of `west build` also shows the order in which _Kconfig_ merges the configuration files. We'll see this in the section about [build types and alternative Kconfig files](#build-types-and-alternative-kconfig-files).
 
-When debugging _Kconfig_ settings, it can be helpful to have a look at the generated output. All _Kconfig_ symbols are merged into a single `zephyr/.config` file located in the build directory, in our case `build/zephyr/.config`. There, we find the following setting:
+When debugging _Kconfig_ settings, it can be helpful to have a look at the generated output. All _Kconfig_ symbols are merged into a single `zephyr/.config` file located in the build directory, in our case, `build/zephyr/.config`. There, we find the following setting:
 
 `build/zephyr/.config`
+
```

Comment:
Note this won't add a new line, it just conforms to a markdown linting rule for ``` braces to be surrounded by new lines

Reply:
I was just using this to write the filename before code blocks. The block following this then uses the correct code block with the corresponding format for its contents.

Reply:
I actually really like you add the filename before the code block! It makes the code blocks easier to understand.

Reply:
Aaaah now I get it: Your markdown linter complained that there was no newline before the code block and you added this newline to silence it (and this newly added newline doesn't actually create an empty new line - like a tongue twister). I'm a bit slow today, sorry.

Reply:
Yep, exactly!

Reply:
My comment probably could have used more precise language